cmd/skaffold/app/cmd/cmd.go

```diff
@@ -155,12 +155,22 @@ func AddRunCommonFlags(cmd *cobra.Command) {
 	cmd.Flags().StringArrayVar(&opts.InsecureRegistries, "insecure-registry", nil, "Target registries for built images which are not secure")
 }
 
-func AddRunDeployFlags(cmd *cobra.Command) {
-	cmd.Flags().BoolVar(&opts.Tail, "tail", false, "Stream logs from deployed objects")
+func AddRunDeployCommonFlags(cmd *cobra.Command) {
 	cmd.Flags().BoolVar(&opts.Force, "force", false, "Recreate kubernetes resources if necessary for deployment (default: false, warning: might cause downtime!)")
 	cmd.Flags().StringArrayVarP(&opts.CustomLabels, "label", "l", nil, "Add custom labels to deployed objects. Set multiple times for multiple labels.")
 }
 
+func AddRunDeployOnlyFlags(cmd *cobra.Command) {
```

Comment:
I'm starting to get really lost in this naming scheme. What does it mean that `RunDeployOnly` and how does it differ from `RunDeployCommon`? I'm starting to think that we should just add `commands=[]string{"run", "dev", "deploy"}` annotation to the flags  somehow...:)

Reply:
yes. its a good idea to add scope annotation to flags.  Its kind of confusing right now. Especially when they call each other.